cloudaeye/README.md

```diff
@@ -0,0 +1,38 @@
+# CloudAEye
+
+## Overview
+
+Kosal from CloudAEye acts as your root cause co-pilot. This integration enables Datadog users to import observability data (logs, metrics, traces) and view automated root cause analyses. Additionally, it identifies similar issues within Jira and related code changes that may be contributing to the incident. To expedite incident identification, Kosal publishes events with a summary and root cause to a Datadog dashboard.
+
+## Setup
+
+### Installation
+
+- Search for CloudAEye in the integrations page.
+- Click on **Install Integration** button.
+- Once installed, click on **Connect Accounts** button which will take you to the [CloudAEye integrations page][1] (Side Navigation Drawer => Integrations => Datadog).
+- Add a Datadog integration. You may have to create your CloudAEye account if you don't have one.
+- Once the keys are populated, your Datadog account will get connected to the CloudAEye.
```

Comment:
This will need to be updated with the steps the user needs to take to set up the integration. You will need to remove the steps for OAuth since your integration does not use OAuth. For example, go to the CloudAEye integration in Datadog, click install, etc. This should be the exact flow the user will take to install the integration.